CHANGELOG.rst

```diff
@@ -27,15 +28,28 @@ Internal Changes
   cert-manager and manually created certificates in both development and production
   contexts.
 
+- **Renku Notebooks:** add session warnings to jupyter server API responses
+  (`#1482 <https://github.com/SwissDataScienceCenter/renku-notebooks/issues/1482>`__)
+  (`b463980 <https://github.com/SwissDataScienceCenter/renku-notebooks/commit/b46398032e6361ef1b69fb4909d2ed87afc583eb>`__)
+
 **Bug Fixes**
 
 - **Knowledge Graph**: Fix for an infinite retry loop while sending certain types of internal events.
   (`#1650 <https://github.com/SwissDataScienceCenter/renku-graph/issues/1650>`_).
 
+- **Renku Notebooks:** parse old server options in request to start sessions
+  (`#1570 <https://github.com/SwissDataScienceCenter/renku-notebooks/issues/1570>`__)
+  (`8b3e5c0 <https://github.com/SwissDataScienceCenter/renku-notebooks/commit/8b3e5c091507446080fd468d84c4bd4b8d134b60>`__)
+
+- **Renku Notebooks:** properly recover LFS files from autosave branches
+  (`#1568 <https://github.com/SwissDataScienceCenter/renku-notebooks/issues/1568>`__)
+  (`8f34b09 <https://github.com/SwissDataScienceCenter/renku-notebooks/commit/8f34b09ab73913bfbba4acbe28b00c53ad576367>`__)
+
```

Comment:
should this be factored out and released as a bugfix for immediate deployment?

Reply:
If possible I agree it would be better

Reply:
@rokroskar the single feature that comes from the notebook service is really simple and only adds a warning field when listing servers. So the whole notebooks release can be deployed immediately.

Reply:
I am not sure how I would isolate this from the KG changes which are already merged in the renku repo.

Reply:
I guess I can have a hotfix branch form which we can tag renku version 0.33.1 which will include only the notebooks release. Is that ok with you @rokroskar?

Reply:
yep - if that's not too much work I think that would be the way to go